Dockerfile.windows

```diff
@@ -1,57 +1,93 @@
-# This file describes the standard way to build Docker, using docker on TP4
+# This file describes the standard way to build Docker, using a docker container on Windows 
+# Server 2016
 #
 # Usage:
 #
-# # Assemble the full dev environment. This is slow the first time.
+# # Assemble the full dev environment. This is slow the first time. Run this from
+# # a directory containing the sources you are validating. For example from 
+# # c:\go\src\github.com\docker\docker
+#
 # docker build -t docker -f Dockerfile.windows .
 #
-# # Mount your source in an interactive container for quick testing:
-# docker run -it -v ${pwd}\bundles:c:/go/src/github.com/docker/docker/bundles docker cmd
 #
+# # Build docker in a container. Run the following from a Windows cmd command prommpt,
+# # replacing c:\built with the directory you want the binaries to be placed on the
+# # host system.
+#
+# docker run --rm -v "c:\built:c:\target" docker sh -c 'cd /c/go/src/github.com/docker/docker; hack/make.sh binary; ec=$?; if [ $ec -eq 0 ]; then robocopy /c/go/src/github.com/docker/docker/bundles/$(cat VERSION)/binary /c/target/binary; fi; exit $ec'
+#
+# Important notes:
+# ---------------
+#
+# Multiple commands in a single powershell RUN command are deliberately not done. This is
+# because PS doesn't have a concept quite like set -e in bash. It would be possible to use 
```

Comment:
OK, I see there is no double quote problem here. It is exact the same behavior running the command from a plain CMD shell.

But lesson learned: `RUN powershell -Command $ErrorActionPreference = 'Stop'; fail; success` is like `RUN fail && success` on Linux